HW1.java

```diff
@@ -0,0 +1,13 @@
+public class HW1 {
+
+        int Year = 2023;
```

Comment:
According to the java code convention it's needed to name variables by lower camel case, so please rename 'Year' to 'year'